### DISCUSSION TITLE
Enabling SSO on superset deployed on ECS fargate

### DISCUSSION THREAD

**Discussion by techgabit** [apache/superset#27944] [Q&A / Help]
I have deployed superset using AWS integration for superset using cloudformation template ... how can I enable SSO on it

**Comment by nigzak** [comment]
Hi
Have you seen this documentation page for OAuth2?
https://superset.apache.org/docs/installation/configuring-superset/#custom-oauth2-configuration 

I don't know if fargate uses OAuth2 - on this page you can find informations about oauth2 and LDAP integration.
More or less you can find my example in another old ticket if you require a full examples (I use OAUTH2 for SSO - but not with fargate) => https://github.com/apache/superset/issues/19510

**Comment by yashgangrades** [comment]
Hi @techgabit  Were you able to figure out how to enable SSO with AWS integration for the Superset CloudFormation template? I have set AUTH_TYPE: AUTH_OAUTH and configured OAUTH_PROVIDERS, but it's still not working. Moreover, the **SupersetInitService** is failing with the error: **Error: Option '--password' requires an argument.**



<br class="Apple-interchange-newline">
</body>
</html>
